### DISCUSSION TITLE
how to access and reach sosetup.log when i type in command it shows acess denied please help me . im wanna see the error that cause the problem in my installation

### DISCUSSION THREAD

**Discussion by hsnroses** [Security-Onion-Solutions/securityonion#11662] [2.4]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8 cores

### RAM

16

### Storage for /

200 gb

### Storage for /nsm

200gb

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

i cant access the web interface even when i check SOC status all services is running well.but befoore that when the installation process is
![WhatsApp Image 2023-10-10 at 17 20 55_ffe137a7](https://github.com/Security-Onion-Solutions/securityonion/assets/139231906/7c96f094-78d1-4665-870f-ca20a276dc57)
 done it shows my installation had a problem like picture shown below. 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/139231906/fe295386-0559-4997-89d8-8bb964bc95e2)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/139231906/c200a5a3-3e28-4462-b19c-77dee29f66ad)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by jaypfe** [comment]
Elevate into root by typing 'sudo su' 
Then try the command again.

**Reply by hsnroses** [reply]
i already access the root by typing sudo su and try command  /root/sosetup.log and still access denied

**Comment by cm-ops** [comment]
Try running `cat /root/sosetup.log`

**Reply by hsnroses** [reply]
thank youu
